### ISSUE TITLE
Allow multiple topic selections

### ISSUE THREAD

**Issue by HarveySWilliams** [sudara/actnational#9]

**Comment by sudara** [comment]
Can you point me to some events where this is the case? 

I'd like to gain an understanding about how frequently this will happen, as well as if the multi-topic events tend to be more "general" in nature (BYOT) vs. specifically 2 or 3 topics max. I'm wondering if we might be better served with a "Multiple Topics" or "General" or "Polytopic" option vs. getting a lot of events with 10+ topics (as they want to be inclusive).

For the 2-3 topic case, we have to make the database logic a bit more complex here, which is no problem. Just best to "prove" it needs to be so.

**Comment by HarveySWilliams** [comment]
Yes, broad justification is that the concept of "intersectionality" is big in progressive movements which leads to many events have multiple themes or covering related issues e.g. Racial Justice + Womyn, Native/Indigenous Rights + Environment.

For most of the events I have created I had to force-choose a topic.

This event should be Native/Indigenous Rights and Environment:
http://www.actnational.org/events/nodapl-march-los-angeles

This event should at least be tagged as both Racial Justice and Immigrant Rights.: http://www.actnational.org/events/sf-protest-against-trump-on-inauguration-day-fight-racism-defend-immigrants

This event should be Racial Justice, Womyn, Immigrant Rights, LGBTI / Queer:
http://www.actnational.org/events/seattle-women-march-against-hate

**Comment by sudara** [comment]
Ok, that looks less like "All The Topics!" and more like: "A couple things we are focusing on."

Would selecting 3-5 items max work?

**Comment by HarveySWilliams** [comment]
Yes, definitely.
